server/routes/organizations/gigs/gig-controller.js

```diff
@@ -3,18 +3,32 @@ module.exports = function(app){
   var Gig = models.Gig;
   var UserGigs = models.UserGigs;
   var User = models.User;
+  var Location = models.Location;
 
   var gigController = {
 
     getGigs: function (req, res){
-      var organizationId = req.organization_id;
-      Gig.getGigs(organizationId).then(function (gigs) {
+      var info = {
```

Comment:
Info seems like a very generic variable name. I didn't really have any hints to the origin of the info object throughout your code until reaching this point.